### PR TITLE
Prepare release 3.4.0

### PR DESCRIPTION
**Pull request by jonasscheid**
Small release 
- Adding NetMHCIIpan 4.2 and 4.3 version 
- Fixing NetMHCIIpan length restrictions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v3.3.2 - 2023-12-15
```
Comment (christopher-mohr) [inline CHANGELOG.md:6]:
```suggestion
## v3.4.0 - 2023-12-12
```

new feature introduced

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
+
+### `Added`
+
+-  [#93](https://github.com/KohlbacherLab/epytope/pull/93) Added NetMHCIIpan version 4.2 interface [#92](https://github.com/KohlbacherLab/epytope/issues/92)
+-  [#96](https://github.com/KohlbacherLab/epytope/pull/93) Added NetMHCIIpan version 4.3 interface [#95](https://github.com/KohlbacherLab/epytope/issues/95)
```
Comment (christopher-mohr) [inline CHANGELOG.md:11]:
```suggestion
-  [#96](https://github.com/KohlbacherLab/epytope/pull/96) Added NetMHCIIpan version 4.3 interface [#95](https://github.com/KohlbacherLab/epytope/issues/95)
```

File: README.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ Currently **epytope** provides implementations of several prediction methods or
   - [NetMHC](https://pubmed.ncbi.nlm.nih.gov/26515819/) 3.0, 3.4, 4.0
   - [NetMHCII](https://pubmed.ncbi.nlm.nih.gov/29315598/) 2.2, 2.3
   - [NetMHCpan](https://pubmed.ncbi.nlm.nih.gov/28978689/) 2.4, 2.8, 3.0, 4.0, 4.1
-  - [NetMHCIIpan](https://pubmed.ncbi.nlm.nih.gov/32406916/) 3.0, 3.1, 4.0, 4.1
+  - [NetMHCIIpan](https://pubmed.ncbi.nlm.nih.gov/32406916/) 3.0, 3.1, 4.0, 4.1, 4.2, 4.3
```
Comment (christopher-mohr) [inline README.md:66]: we might want to check at some point for additional publications here

File: setup.py [hunk 4]
```diff
@@ -21,7 +21,7 @@
     name='epytope',
 
     # Version:
-    version='3.3.1',
+    version='3.3.2',
```
Comment (christopher-mohr) [inline setup.py:24]:
```suggestion
    version='3.4.0',
```

File: setup.py [hunk 5]
```diff
@@ -35,8 +35,8 @@
     author_email='benjamin.schubert@helmholtz-muenchen.de, walzer@ebi.ac.uk, christopher.mohr@uni-tuebingen.de, leon.kuchenbecker@uni-tuebingen.de ',
```
Comment (christopher-mohr) [inline setup.py:35]:
```suggestion
    author_email='benjamin.schubert@helmholtz-muenchen.de, walzer@ebi.ac.uk, contact.cmohr@gmail.com, leon.kuchenbecker@uni-tuebingen.de ',
```

**Review by christopher-mohr — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jonasscheid) [reply README.md:66]: Good point. There is a new publication on the 4.3 classII predictor. I'll update to the latest publication

**Review by christopher-mohr — APPROVED** [review]
LGTM

**Pull request merged**